src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -891,31 +841,23 @@ public void RemoveOrganelle(Hex hex)
     {
         int q = hex.Q;
         int r = hex.R;
-
         switch (Symmetry)
         {
             case MicrobeSymmetry.None:
-            {
```

Comment:
While both brace styles (with and without) are allowed, please don't remove the braces just for the sake of removing them.